### PR TITLE
Ronglu edit

### PR DESCRIPTION
**Pull request by ronglums**
Moved the Extension building instructions into a separate file.

### CODE REVIEW

File: Documentation/Getting started with IntelliSense configuration.md [hunk 1]
```diff
@@ -1,17 +1,3 @@
-## How to build and debug the Microsoft CppTools Extension
-
```
Comment (sean-mcmanus) [inline Documentation/Getting started with IntelliSense configuration.md:2]: I think this version of the file should be used. It looks like the new file has the old contents (i.e. I'm getting 404 not found in the links)?

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Documentation/Getting started with IntelliSense configuration.md (hunk 1 above)
Comment (ronglums) [reply Documentation/Getting started with IntelliSense configuration.md:2]: I didn't touch the content of the file...which link isn't working?

Comment (sean-mcmanus) [reply Documentation/Getting started with IntelliSense configuration.md:2]: The contents were changed in master. The code you deleted doesn't match the code you added. The last link doesn't work.

Comment (sean-mcmanus) [reply Documentation/Getting started with IntelliSense configuration.md:2]: e.g. The `** **` are missing.

Comment (ronglums) [reply Documentation/Getting started with IntelliSense configuration.md:2]: ah, didn't realize the content has been modified by someone else in master. i'll update.

**Comment by ronglums** [comment]
Leave a comment

**Pull request closed without merging**